website/blog/2022-03-14-an-update-on-the-new-architecture-rollout.md

```diff
@@ -0,0 +1,114 @@
+
+### The 3rd-party Libraries Ecosystem
+
+We could not migrate to the New Architecture without the full support of **3rd-party libraries author & maintainers**.
+
+We understand how this can be a tedious process, and we understand the importance of supporting users on **both** old and New Architecture. Over the next months, we will focus on supporting our library developers to help them migrate over.
+
+If you’re a **library developer**, [we invite you to post an update](https://github.com/reactwg/react-native-new-architecture/discussions/categories/libraries) in the New Architecture working group with the **status of your libraries**. This will help you attract early adopters and us to understand if any library is facing a blocker.
+
+If instead you’re a **library user**, you can [post a message here](https://github.com/reactwg/react-native-new-architecture/discussions/6) to request a migration of a library. If we identify a library that becomes a blocker for a number of users, we will try to reach out to the maintainer and understand why they haven’t migrated yet.
+
+Finally, we would like to give a shout out to Software Mansion for releasing a new version of [`react-native-screens`](https://github.com/software-mansion/react-native-screens), which has support for both architectures. Moreover, they published a blog-post ([Introducing Fabric to react-native-screens](https://blog.swmansion.com/introducing-fabric-to-react-native-screens-fd17bf18858e)) where they **tell their migration story**. We hope you will find this story inspiring and useful to tackle your migration.
+
+### Releases
+
+Work on the 0.68 pre-release has realized much of [the improved release process we had defined last half](/blog/2022/01/19/version-067#improvements-to-release-process).
+
+We’re happy to share that with 0.68 we were able to:
+
+- Successfully onboard release work to an internal rotation. Much of this is supported by [improved documentation](https://github.com/facebook/react-native/wiki/releases) on the release process which will reduce the bus factor of the release process.
+- Initiated discussions with partners to support a [Copilot rotation](/contributing/release-roles-responsibilities#release-role-2--release-copilot). We hope this effort will improve transparency of the process and inform our partners where to invest to support React Native releases and eco-system.
+- [Onboarded several Release Supporters and Testers from the community](https://github.com/reactwg/react-native-releases/discussions/11). We had put a call-out for help last half and so many folks stepped up! The feedback from our testers and supporters have **helped us fix crucial bugs** and regressions, especially around the new architecture, for the upcoming release. Thank you to everyone who signed up and tested out the release! If you’d like to join as a release tester or supporter, please sign up here!
```

Comment:
```suggestion
- [Onboarded several Release Supporters and Testers from the community](https://github.com/reactwg/react-native-releases/discussions/11). We had put a call-out for help last half and so many folks stepped up! The feedback from our testers and supporters have **helped us fix crucial bugs** and regressions, especially around the new architecture, for the upcoming release. Thank you to everyone who signed up and tested out the release! If you’d like to join as a release tester or supporter, [please sign up here](ADD ME)!
```
Link is missing

Reply:
Removing for now as the sign up is on hold for now (cc @lunaleaps).

Reply:
https://forms.gle/fPuPE1MZRDGWNqpd6